cmd/up.go

```diff
@@ -105,7 +105,8 @@ func Up() *cobra.Command {
 			if syncthingUpgradeAvailable() {
 				fmt.Println("Installing dependencies...")
 				if err := downloadSyncthing(); err != nil {
-					return fmt.Errorf("couldn't download syncthing, please try again")
+					log.Yellow("couldn't download syncthing, please try again later")
```

Comment:
@pchico83 what do you think about this part?

Reply:
does it fail if a previous version is not downloaded?

Reply:
> does it fail if a previous version is not downloaded?

It will fail further down when we try to launch `syncthing`

Reply:
I would fail here in that case